app/views/games/show.html.erb

```diff
@@ -13,15 +13,15 @@
 </div>
 
 <script>
-  var whitePieceHtml = {
+  const whitePieceHtml = {
```

Comment:
`const` is awesome, however we should be aware that using it limits us to IE 11 and newer (Edge) and Safari 10.  Safari 9 technically supports `const`, but only if strict mode is disabled, and *many* JavaScript transpilers enable strict mode (it's generally a good idea).  If we're okay with those restrictions, then this change is fine.  The IE restriction is fine, the number of users on IE10 is incredibly small since it's an autoupdate to IE 11 / Edge.  The Safari one is a bit more restrictive, since older OS X version will be affected.  However, since our app is small, I don't think we need to worry about that, and this is a good place to practice with `const`.

If you get further into JS transpilers (`babel`) or bundlers (`webpack`) at some point in your dev career, then you can use `let` / `const` everywhere and just let the transpiler downgrade it to `var` as necessary.

👍